Gemfile

```diff
@@ -8,7 +8,9 @@ ruby '3.0.2'
 gem 'rails', '~> 6.1.5'
 
 gem 'bootsnap', '>= 1.4.4', require: false # https://rubygems.org/gems/bootsnap
+gem 'discard', '~> 1.2'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rails
+gem "image_processing", ">= 1.2"
```

Comment:
This gem is not a requirement of the ticket, please check [this tutorial](https://codigofacilito.com/articulos/active-storage) about how to configure active storage in the project.